db/model/event.go

```diff
@@ -14,8 +14,9 @@ type Event struct {
 	ChallengerAddress string       `gorm:"NOT NULL"`
 	Height            uint64       `gorm:"NOT NULL;"`
 	Status            EventStatus  `gorm:"NOT NULL;index:idx_status"`
-	VerifyResult      VerifyResult `gorm:"NOT NULL;"`
+	VerifyResult      VerifyResult `gorm:"NOT NULL;index:idx_verify_result"`
 	CreatedTime       int64        `gorm:"NOT NULL"`
+	ExpiredHeight     uint64       `gorm:"NOT NULL;index:expired_height"`
```

Comment:
Status `EnoughVotesCollected` is not used now.

Reply:
fixed